DEPS

```diff
@@ -243,7 +243,7 @@ hooks = [
     'name': 'download_dml_unpkg',
     'pattern': '.',
     'condition': 'checkout_win',
-    'action': ['python3', 'src/webnn/native/dml/deps/script/download_dml.py'],
+    'action': ['python3', 'src/webnn/native/dmlx/deps/script/download_dml.py'],
```

Comment:
keep the name `dml `

Reply:
While I need the `dml` as folder name for new backend. I want to match the folder name with the backend name.@fujunwei

Reply:
Yes, both of dml and dmlx need to download the binary, so keep it in `src/webnn/native/dml/deps/script/download_dml.py`

Reply:
> Yes, both of dml and dmlx need to download the binary, so keep it in `src/webnn/native/dml/deps/script/download_dml.py`

Agree, I understood, I will move the download_dml.py under src/webnn/native/dml/ when the new backend is submitted, is that OK? @fujunwei

Reply:
Move into third_party. @fujunwei